elements/pl-checkbox/pl-checkbox.css

```diff
@@ -7,3 +7,23 @@
     min-width: 1.25em;
     overflow-wrap: normal;
 }
+
+.pl-checkbox-answer-feedback {
+    border-left: 0.25rem solid rgba(0, 0, 0, 0.4);
+    background-color: rgba(0, 0, 0, 0.03);
+    padding: 0.25rem 0.5rem;
+    margin-left: 0.75rem;
+    margin-right: 0.5rem;
+    border-radius: 0.2rem;
+}
+
+.pl-checkbox-submission-feedback {
+    border-left: 0.25rem solid rgba(0, 0, 0, 0.4);
+    background-color: rgba(0, 0, 0, 0.03);
+    padding: 0.25rem 0.5rem;
+    margin-left: 0.75rem;
+    margin-right: 0.5rem;
+    margin-top: 0.25rem;
+    margin-bottom: 0.25rem;
+    border-radius: 0.2rem;
+}
```

Comment:
AFAICT, these are virtually the same (expect the latter has additional top/bottom margins. Consider collapsing the shared bits into a single `.pl-checkbox-feedback` class.

Separately, we generally prefer to use Bootstrap classes to style things - this means applying classes like `ml-2`, `rounded`, etc. instead of using bespoke styles like these in a CSS file.